# Do not use affinity (as it already should be set by the queue system) with proscan utils
# July 15th 2021 by B. Hajgato (UGent)
--- interproscan-5.52-86.0/bin/prosite/ps_scan.pl.org	2021-06-02 16:33:37.000000000 +0200
+++ interproscan-5.52-86.0/bin/prosite/ps_scan.pl	2021-07-15 16:00:51.589475628 +0200
@@ -1730,7 +1730,7 @@
         if ($level_arg eq 0) { # yes default scan l=0 is internally run at -1 (so that postprocessing could consider weak matches)
             $level_arg = -1;
         }
-        @pre_command  = $use_pfscan_v3 ? "$PFSCAN --matrix-only -o4 -L$level_arg $PROSITE" : "$PFSCAN -flxz -v -C$level_arg";
+        @pre_command  = $use_pfscan_v3 ? "$PFSCAN --no-affinity --matrix-only -o4 -L$level_arg $PROSITE" : "$PFSCAN -flxz -v -C$level_arg";
         @post_command = $use_pfscan_v3 ? "" : $PROSITE;
     }
     my $out;
--- interproscan-5.52-86.0/bin/prosite/runprosite.py.org	2021-06-02 16:33:37.000000000 +0200
+++ interproscan-5.52-86.0/bin/prosite/runprosite.py	2021-07-15 16:00:25.908539401 +0200
@@ -43,7 +43,7 @@
         binary_command = arg_list[4]
         binary_switches = arg_list[5:]
 
-        pfsearch_cmd_switches = ' '.join(binary_switches)
+        pfsearch_cmd_switches = ' '.join(['--no-affinity'] + binary_switches)
         #print pfsearch_cmd
 
         #print (binary_switches)
